src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/lecture-unit-management.component.ts

```diff
@@ -51,20 +58,27 @@ export class LectureUnitManagementComponent implements OnInit, OnDestroy {
     // Icons
     faTrash = faTrash;
     faPencilAlt = faPencilAlt;
+    sendToIris = faFileExport;
+    resendToIris = faRepeat;
+    done = faCheckCircle;
+    loading = faSpinner;
```

Comment:
Please add `readonly` to all these attributes.
Additionally I haven't seen any other place in Artemis where we use other variable names for the icons, so I would stick to `faFileExport` and so on as names